reference/components/document-manager/fixtures.rst

```diff
@@ -7,74 +7,122 @@ you to load static data into your content repository.
 Getting Started
 ---------------
 
-Shown below is the simple data fixtures:
+Shown below is an example that creates a simple data fixture.
+
+**Note**
+    - The class name MUST end with `Fixture` for it to be recognized
+    - The class MUST be placed in `<your bundle>/DataFixtures/Document` in order
+      for it to be loaded automatically.
 
 .. code-block:: php
 
     <?php
-    // YourBundle/DataFixtures/Document/SomeFixture.php
-
     namespace YourBundle\DataFixtures\Document;
 
     use Sulu\Component\DocumentManager\DocumentManager;
     use Sulu\Bundle\DocumentManagerBundle\DataFixtures\DocumentFixtureInterface;
-    use Sulu\Component\Content\Document\WorkflowStage;
+    use Sulu\Component\DocumentManager\Exception\MetadataNotFoundException;
 
     class SomeFixture implements DocumentFixtureInterface
     {
+        /**
+         * Simple local string with two chars.
+         */
+        const LOCALE = 'en';
+
+        /**
+         * All fixtures will be sorted in regards of the returned integer. This
+         * "weight" will let a fixture run later if the integer is higher.
+         *
+         * @return int
+         */
         public function getOrder()
         {
             return 10;
         }
 
+        /**
+         * Load fixtures.
+         *
+         * Use the document manager to create and save fixtures.
+         * Be sure to call DocumentManager#save() when you are done.
```

Comment:
A `save` method does not exist, I guess you mean `flush` :-)

Reply:
That is the comment from your interface :P

I had asked Alex already and thought I had changed it -_-

interface DocumentFixtureInterface
{
    /**
     * Load fixtures.
     *
     * Use the document manager to create and save fixtures.
     * Be sure to call DocumentManager#save() when you are done.
     *
     * @param DocumentManager
     */
    public function load(DocumentManager $documentManager);